/razor_node:
  ros__parameters:
    port: /dev/ttyUSB0
    topic: imu
    frame_id: base_imu_link
    
    # Calibration
    accel_x_min: -250.0
    accel_x_max: 250.0
    accel_y_min: -250.0
    accel_y_max: 250.0
    accel_z_min: -250.0
    accel_z_max: 250.0
    
    magn_x_min: -600.0
    magn_x_max: 600.0
    magn_y_min: -600.0
    magn_y_max: 600.0
    magn_z_min: -600.0
    magn_z_max: 600.0
    
    calibration_magn_use_extended: false
    magn_ellipsoid_center: [0.0, 0.0, 0.0]
    # 3x3 Matrix Flattened: [row1, row2, row3]
    magn_ellipsoid_transform: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    
    imu_yaw_calibration: 0.0
    
    gyro_average_offset_x: 0.0
    gyro_average_offset_y: 0.0
    gyro_average_offset_z: 0.0
